db/seeds/prisons/CFI-cardiff.yml

```diff
@@ -6,31 +6,15 @@ address:
 - Cardiff
 - CF24 0UG
 booking_window: 14
-canned_responses: true
```

Comment:
As I said in the commit message, I wanted the prison I was using for manual testing to be completely up-to-date.  Strictly speaking, it wasn't needed, and I can drop it out if you'd prefer.